### PR TITLE
[All] Scrub Office 365 platforms

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
See related PR https://github.com/OfficeDev/office-js-docs-pr/pull/994.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3878bb1](https://github.com/OfficeDev/office-js-docs-pr/pull/995/commits/3878bb1ba4a641b8e797762311fc255ed0736f9c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/design/add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-office-add-ins-for-the-ipad.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/develop-office-add-ins-for-the-ipad.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-troubleshooting.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-core-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/index.yml) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/overview/office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/overview/office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/action.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/action.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/desktopformfactor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/desktopformfactor.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/officetab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/officetab.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/add-in-commands-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/add-in-commands-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/dialog-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/dialog-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/office-add-in-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/office-add-in-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/outlook-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/outlook-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |
[docs/word/word-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/word/word-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905090027129432-995%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dJ4vp6H9yvrr%2FNc9jSJcsSTnMzdqjrNNNsMwSG67APk%3D&st=2019-05-09T00%3A24%3A03Z&se=2019-06-09T00%3A29%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[989cc32](https://github.com/OfficeDev/office-js-docs-pr/pull/995/commits/989cc322ad98d24cf9b4f7aff2a3d20a4d4f7fdc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/design/add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-office-add-ins-for-the-ipad.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/develop-office-add-ins-for-the-ipad.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-troubleshooting.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-core-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/index.yml) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/overview/office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/overview/office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/action.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/action.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/desktopformfactor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/desktopformfactor.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/officetab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/officetab.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/add-in-commands-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/add-in-commands-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/dialog-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/dialog-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/office-add-in-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/office-add-in-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/outlook-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/outlook-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |
[docs/word/word-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/word/word-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905091853529968-995%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=CBIFY4ZxbQjy8s7aOj2ZSNZh%2FUfa2zQo7o8%2F0omItXs%3D&st=2019-05-09T18%3A50%3A31Z&se=2019-06-09T18%3A55%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-best-practices.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ The following sample shows the JSON metadata that corresponds to the functions d
 
 ## Additional considerations
 
-Avoid accessing the Document Object Model (DOM) directly or indirectly (for example, using jQuery) from your custom function. On Excel for Windows, where custom functions use the [JavaScript runtime](custom-functions-runtime.md), custom functions cannot access the DOM.
+Avoid accessing the Document Object Model (DOM) directly or indirectly (for example, using jQuery) from your custom function. In Excel on Windows, where custom functions use the [JavaScript runtime](custom-functions-runtime.md), custom functions cannot access the DOM.
```
Comment (mscharlock) [inline docs/excel/custom-functions-best-practices.md]: Thanks for fixing these, this is one of those I'll have to add to my list - I didn't realize it's FOR mac/ipad but ON windows

File: docs/testing/attach-debugger-from-task-pane.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 description: ''
-ms.date: 03/19/2019
+ms.date: 05/08/2019
 localization_priority: Priority
 ---
 
 # Attach a debugger from the task pane
 
-In Office 2016 for Windows, Build 77xx.xxxx or later, you can attach the debugger from the task pane. The attach debugger feature will directly attach the debugger to the correct Internet Explorer process for you. You can attach a debugger regardless of whether you are using Yeoman Generator, Visual Studio Code, node.js, Angular, or another tool. 
+In Office 2016 on Windows, Build 77xx.xxxx or later, you can attach the debugger from the task pane. The attach debugger feature will directly attach the debugger to the correct Internet Explorer process for you. You can attach a debugger regardless of whether you are using Yeoman Generator, Visual Studio Code, node.js, Angular, or another tool.
```
Comment (mscharlock) [inline docs/testing/attach-debugger-from-task-pane.md]:
```suggestion
In Office 2016 on Windows, Build 77xx.xxxx or later, you can attach the debugger from the task pane. The attach debugger feature will directly attach the debugger to the correct Internet Explorer process for you. You can attach a debugger regardless of whether you are using Yeoman Generator, Visual Studio Code, Node.js, Angular, or another tool. 
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5994b0e](https://github.com/OfficeDev/office-js-docs-pr/pull/995/commits/5994b0e278d6c0c28a765eee5fc6d5838488807d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/design/add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-office-add-ins-for-the-ipad.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/develop-office-add-ins-for-the-ipad.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/custom-functions-troubleshooting.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-core-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/index.yml) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/overview/office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/overview/office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/action.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/action.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/desktopformfactor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/desktopformfactor.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/officetab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/manifest/officetab.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/add-in-commands-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/add-in-commands-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/dialog-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/dialog-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/office-add-in-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/office-add-in-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/outlook-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/outlook-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |
[docs/word/word-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-scrub-platforms/docs/word/word-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905091916440987-995%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2p8VtoIyCUchjxdN2Ic%2FffiuTA1hzVIR0ItOoSYZKZs%3D&st=2019-05-09T19%3A14%3A17Z&se=2019-06-09T19%3A19%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**